wazo_webhookd/plugins/subscription/service.py

```diff
@@ -22,11 +22,15 @@
 
 class SubscriptionService(object):
 
+    # NOTE(sileht): We use a class variable to ensure all instances
+    # SubscriptionService will share the same pubsub object.
+
+    pubsub = Pubsub()
```

Comment:
Good idea :+1: 
Does it intent to fix something or it is a refactor/better approach in case we need to instantiate more service?

Reply:
It does, push mobile plugin instantiates another SubscriptionService() to create subscriptions on the flight when a mobile is registered in wazo.

When push mobile was creating subscription, SubscriptionBusEventHandler was not receiving the informations and was not creating the required kombu Queue/Binding/Consumer and was never received the expected events until we restart websocketd.

By sharing the PubSub() between all instances of SubscriptionService(), now SubscriptionBusEventHandler() always creates kombu resources when expected.

Reply:
I will add some of this in the NOTE

Reply:
That's ok. It's  because I only see one instance in the code and I didn't think to external plugin. No really need to be commented